lib/tasks/buildrequires.rake

```diff
@@ -0,0 +1,55 @@
+#--
+# Yast rake
+#
+# Copyright (C) 2017 SUSE LLC
+#   This library is free software; you can redistribute it and/or modify
+# it only under the terms of version 2.1 of the GNU Lesser General Public
+# License as published by the Free Software Foundation.
+#
+#   This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+# details.
+#
+#   You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#++
+
+require "shellwords"
+require "open3"
+
+namespace :buildrequires do
```

Comment:
I just prefer to call it `:build_dependencies` buildrequires is rpm specific. Otherwise it looks good.

Reply:
OK, makes sense.